.github/workflows/release.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
       - run: make setup
       - run: make build TOPOLVM_VERSION=${{ steps.check_version.outputs.version }}
```

Comment:
`lvmd` is stil needed as standalone binary so this line is not removed yet.
I'll make new PullRequest after https://github.com/topolvm/topolvm/issues/195 is implemented

Reply:
Yeah, it is still needed.
Moreover, after #195 is done, someone (including ours) need to migrate to use daemonset,
there is no need to rush to delete lines.